### DISCUSSION TITLE
Beta channel / master builds?

### DISCUSSION THREAD

**Discussion by harshad1** [gsantner/markor#1718] [General] [answered]
@gsantner Would it be possible to create a beta channel for markor. Something like a weekly release build of the master branch, downloadable here.

It would be even nicer if this was available as an F-Droid repository, but I have no idea if that can be hosted on github.

**Comment by gsantner** [comment, marked as answer]
You can download a APK from the actions tab. So, from any current commit, so even those from current in-development-branches and pull-requests.

I dont plan a second release trail. People coming here, or asking me by email directly ... already have troubles telling the version with just the normal release. I don't want to imagine how it is when adding another point to the matrix.
